src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -487,15 +532,27 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
                 returnValue = new Ammo.btBoxShape(this._tmpAmmoVectorA);
                 break;
             case PhysicsImpostor.MeshImpostor:

```

Comment:
I would prefer to add a new Impostor type like ConvexHullImpostor

@TrevorDev  thoughts?

Reply:
I added ConvexHullImpostor type support for Ammo.btConvexHullShape